src/routes/healthcheck.ts

```diff
@@ -6,7 +6,9 @@ const CMR_INGEST_HEALTH = `${CMR_LB_URL}/ingest/health`;
 const CMR_SEARCH_HEALTH = `${CMR_LB_URL}/search/health`;
 
 export const healthcheckHandler = async (_req: Request, res: Response) => {
+  console.debug(`GET ${CMR_INGEST_HEALTH}`);
```

Comment:
I don't see any issue with adding more logs I need to check if these will actually print on cloud-watch though, We do already have quite a few logs in place for when graphql calls go out which makes the Search health one sort of redundant if search or ingest health are down its going to be easy to correlate to a CMR DSE